hysds/user_rules_job.py

```diff
@@ -19,12 +19,12 @@
 from elasticsearch import ElasticsearchException
 
 JOBS_ES_URL = app.conf.JOBS_ES_URL  # ES
-STATUS_ALIAS = app.conf.STATUS_ALIAS
```

Comment:
@DustinKLo: did a quick sanity check on the other core repos and found this:
```
$ cd lightweight-jobs
$ grep -r STATUS_ALIAS .
./purge.py:        es_index = app.conf["STATUS_ALIAS"]
./retry.py:STATUS_ALIAS = app.conf["STATUS_ALIAS"]
./retry.py:    return mozart_es.search(index=STATUS_ALIAS, body=query_json)
./lib/get_component_configuration.py:        query_idx = app.conf["STATUS_ALIAS"]
./notify_by_email.py:        index = app.conf["STATUS_ALIAS"]
```
Should there also be a PR on lightweight-jobs?

Reply:
@pymonger i think `retry.py` and `purge.py` would need to be updated
`get_component_configuration.py` isn't used anywhere and `notify_by_email.py`